app/src/main/res/values-v23/theme_toy.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="ThemeToy.Light">
```

Comment:
This style does nothing for now but keeps a hint that these color will eventually need a Light/Dark design in the future for API 23 & above, because the statusbar style does not look good when we change background theme color to the dark one.